test/common/lcov.py

```diff
@@ -482,9 +482,13 @@ def limit_comments(comments: list[dict[str, str]]) -> tuple[list[dict[str, str]]
     if len(comments) <= 4:
         return comments, ""
 
-    ta_url = os.environ.get("TEST_ATTACHMENTS_URL", None)
-    coverage_report = f"{ta_url}/Coverage/index.html"
-    report_link_comment = f"There are more than 10 code coverage comments, see the full report at {coverage_report}"
+    log_url = os.environ.get("COCKPIT_CI_LOG_URL", None)
+    if log_url is not None:
+        base_url = log_url.replace("log.html", "")
```

Comment:
I'd prefer to make this a bit more natural using `os.path.dirname(log_url)`

That also fixes the double slash in the generated URL: https://cockpit-logs.us-east-1.linodeobjects.com/pull-22180-9baec84a-20250707-123857-fedora-42-devel//Coverage/index.html